content/30.abstracting_simulation_types.md

```diff
@@ -86,7 +86,26 @@ Scheme of the AMR structure used to estimate the gradient of a quantity in the c
 
 ### SPH Analysis
 
-Smoothed Particle Hydrodynamics (SPH) is **MJT: We need a very brief explanation of SPH here!**
+Smoothed Particle Hydrodynamics (SPH) is a commonly-used method for solving equations of hydrodynamics in astrophysics (as well as many other fields!) from a lagrangian perspective.
+This provides many advantages over grid-based discretizations, but also poses somewhat different challenges for analysis and visualization.
+While a full description of SPH is outside the scope of this paper, there are a handful of crucial and important pieces of information that we will review.
+For more information, we refer interested readers to this comprehensive review of Smoothed Particle Hydrodynamics by Volker Springel [@doi:10.1146/annurev-astro-081309-130914] or to the SPLASH method paper by Daniel Price [@doi:10.1071/AS07022].
+SPH defines field quantities *at* a set of moving points, and field values themselves can be computed by integrating over all the elements in the domain using a special-purpose kernel.
+Formally, this is represented as:
+$$
+A(\mathbf{r}) = \int A(\mathbf{r}')W(|\mathbf{r} - \mathbf{r}'|,h)\mathrm{d}V(\mathbf{r}')
+$$
+This is then reduced to a sum over the particles (the discretization points):
+$$
+A(\mathbf{r}) = \sum_j V_j A_j W(|\mathbf{r} - \mathbf{r}'|, h)
+$$
+In these equations, $A$ is the field, W is the weighting function (the 'kernel') and $h$ is the smoothing length.
+(One important note is that for various reasons, the smoothing length and the 'half' smoothing length are often used in different contexts in SPH; this is something to keep in mind, as variable names such as `hsml` may in fact refer to 'half-smoothing length.')
+By choosing a smoothing kernel that terminates outside of a given radius, this can be restricted to a finite subset of the number of particles in the simulation, and provide a natural adaptivity in resolution.
+There are other formulations of SPH as well that confine the integral to a fixed number of particles, that evaluate based on pressure criteria, and so forth.
```

Comment:
I would just mention scatter/gather here in the context of defining how you choose that fixed number of particles (would also be good to use the word "neighbors" somewhere, and probably mention that conventionally this is 32).

Reply:
So something like "In general, there are two approaches to defining this finite set of particles used to interpolate the field at the point of interest (often referred to as neighbors, of which conventionally there are 32). The first is the "gather method," wherein you calculate the smoothing length at the point of interest and find all particles within that distance. The second is the "scatter method" wherein you find all the particles whose smoothing lengths overlap the point of interest." 

Maybe there's a way to condense that